recipe/install.bat

```diff
@@ -17,6 +17,8 @@ cmake -G "Ninja" ^
       -D HDF5_BUILD_HL_LIB:BOOL=ON ^
       -D HDF5_BUILD_TOOLS:BOOL=ON ^
       -D HDF5_ENABLE_Z_LIB_SUPPORT:BOOL=ON ^
+      -D SZIP_LIBRARY:FILEPATH=%LIBRARY_PREFIX%\libsz.so ^
```

Comment:
Is this line correct. so files are not for windows. Maybe you mean .lib

Reply:
Sorry, I have no idea, this was my best guess as a person that doesn't have access to Windows. (I also don't know if `\lib` is necessary.)

Reply:
Try to specify with \lib